app/views/welcome/_cards.html.erb

```diff
@@ -1,7 +1,9 @@
 <h3 class="title"><%= t("welcome.cards.title") %></h3>
 
-<div class="row" data-equalizer data-equalizer-on="medium">
-  <% @cards.each do |card| %>
-    <%= render "card", card: card %>
-  <% end %>
+<div class="row">
+  <div class="cards-container">
+    <% @cards.each do |card| %>
+      <%= render "card", card: card %>
+    <% end %>
+  </div>
 </div>
```

Comment:
Maybe we could also extract this block into a partial in either `cards/_cards` or `shared/_cards` :thinking:. What do you think?